wolfcrypt/src/asn.c

```diff
@@ -11110,8 +11125,8 @@ int wc_EccPublicKeyDecode(const byte* input, word32* inOutIdx,
         }
         if (ret == 0) {
             SkipObjectId(input, inOutIdx, inSz);
-            ret = ASNToHexString(input, inOutIdx, (char**)&curve->prime, inSz,
-                                            key->heap, DYNAMIC_TYPE_ECC_BUFFER);
+            ret = ASNToHexString(input, inOutIdx, &len, (char**)&curve->prime,
```

Comment:
Why are we passing in &len and then not using it?

Reply:
There was spots where the String was being set in ASNToHexString and there was no need to copy the String to other buffers later on in the code. Reviewing it again it may be cleaner to not add the &len argument and to instead use XSTRLEN when needed like what was done on line 11132.